### DISCUSSION TITLE
胡安会在9月份来中国吗?

### DISCUSSION THREAD

**Discussion by xzsean666** [filecoin-project/community-china#132] [新手上路]
我这边听说了好多消息,听说胡安要来成都,太让我激动了.
Hi, I just heard a news ,that Juan Benet is going to join events in ChengDu city in china September ,is that ture?,Im very excited about it .

**Comment by yhhwpp** [comment]
都在谣传不知真假

**Comment by BBQFIL** [comment]
这怎么还一股子伦敦腔？

**Comment by zhaobo443** [comment]
FIL联合封装，用户提供质押币，公司提供设备和运维，收益60％,公司扣出20％服务费和20%设备费，合同期1080天，540天质押币退还，t3天封装。可实地考察。V:zhaobo9308,另接代封装业务，矿池搭建和运维服务，价格美丽，提供教学指导
